frontend/routes/package/(_components)/PackageHeader.tsx

```diff
@@ -185,7 +185,9 @@ export function PackageHeader(
                       10,
                     )}
                 >
-                  {twas(new Date(selectedVersion.createdAt))}
+                  {`${
+                    twas(new Date(selectedVersion.createdAt))
+                  } (v${pkg.latestVersion})`}
```

Comment:
updated to use selected version instead of latest version, I forgot this block was used in any selected version. Since the other elements nearby (jsr score and compat) reference the current version I think it may be nice to include a version reference next to that timeframe, though maybe it would be better as part of the "Published" header? 

Like 
```
v14.0.2 Published
3 months ago
```
wdyt?

Reply:
hm, that could work, but i feel like its implied if you are on the specific version's page